src/utils/starknet_compilation.py

```diff
@@ -72,3 +73,18 @@ def compile_contract(
         )
         assert isinstance(assembled, ContractDefinition)
         return assembled
+
+    class AbiElement(TypedDict):
```

Comment:
Would prefer just using cairo internal object (dict?)

Reply:
> it's very cairo implementation dependent,
Yet, we are messing with its internals in various places. I created this type to avoid debugging this object every time we do something with it.

I'll make this type more obscure by leaving fields that are actually used and add proper comment to this type.